core/controllers/acl_decorators.py

```diff
@@ -2753,6 +2753,82 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
+def can_reviewer_or_suggester_update_suggestions(handler):
```

Comment:
I'm not sure this name is much clearer. Unless I'm missing something, the decorator isn't actually checking whether the user is a reviewer or suggester, and it still seems specific to translations.

However, if this is meant to be the generic check for updating suggestions then perhaps something generic like your previous name is actually preferable: ``can_update_suggestions``. My earlier comment was me trying to figure out how generic you intend this function to be, and per other comment threads it actually seems like it's meant to be used for any suggestion types eventually.

Reply:
Updated the name back as `can_update_suggestions`